src/player.cpp

```diff
@@ -10417,14 +10417,12 @@ bool player::wield(item* it, bool autodrop)
     // than a skilled player with a holster.
     // There is an additional penalty when wielding items from the inventory whilst currently grabbed.
 
+    mv += it->volume() * VOLUME_MOVE_COST;
```

Comment:
`it->volume() * VOLUME_MOVE_COST` is reused a lot.
It would be nice if it was wrapped a function. That would allow easy modification, for example by hand encumbrance or mutations.